assets/video-section.css

```diff
@@ -26,3 +27,9 @@
   width: 100%;
   height: 100%;
 }
+
+.video-section__media video {
+  width: 100%;
+  height: 100%;
+  position: absolute;
+}
```

Comment:
I think we also need to add a rule so that when the section is full width  the video itself doesn't get some border radius applied: 
![](https://screenshot.click/14-27-2mzc5-yy87b.gif)

Reply:
Good catch, needed to expand the existing selectors more https://github.com/Shopify/dawn/pull/2248/commits/3228fe56c4fb669cc4e8a15be01bf7abb63e30e5